wordpress-mobile/WordPress-Android
# Replace StringUtils.compare with inline code

Fixes #7376 

To test:
No steps available. That said, the crash seems to happen predominantly on Xiaomi devices running Android below v7. So, if you happen to have such a device, try to go to Mysite > Site settings > Timezone and see if the app crashes or not.
